datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -311,3 +311,26 @@ def get_tox_env_python_version(env):
     match = re.match(PYTHON_MAJOR_PATTERN, env)
     if match:
         return int(match.group(1))
+
+
+def process_checks_option(checks, source=None):
```

Comment:
I'd get rid of this and simply modify https://github.com/DataDog/integrations-core/blob/master/.azure-pipelines/templates/run-validations.yml to add check arg

Reply:
But with this change you can run `ddev validate all changed` locally.  And for CI we can add `changed` which should auto-configure to the appropriate checks for PRs.